package.json

```diff
@@ -40,6 +40,7 @@
   "devDependencies": {
     "browserify": "^13.0.0",
     "d3": "3.5",
+    "lodash.result": "^4.4.0",
```

Comment:
@esjewett 
> What would it look like to implement this without using lodash?

.

> I just checked it out and lodash.result is small and well-encapsulated

```lodash.result``` is quite robust while still being relatively small. If necessary, an internal implementation could trade off robustness for a smaller footprint.